### PR TITLE
Implemented silent waypoints for Android.

### PR DESCRIPTION
**Pull request by neildunlop**
Implemented changes so that the 'isSilent' property is respected by the navigation UI.
Silent waypoints are not announced by the navigator.
Waypoints that are not silent have their name announced by the navigator.

### CODE REVIEW

File: android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/models/SimpleWaypoint.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.eopeter.flutter_mapbox_navigation.models
+
+import com.google.gson.annotations.SerializedName
+import java.io.Serializable
+
+/**
+ * A simple class to capture how most people will express a waypoint.
+ */
+data class SimpleWaypoint (
```
Comment (eopeter) [inline android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/models/SimpleWaypoint.kt:9]: Why couldn't this be called `Waypoint`?

File: .idea/misc.xml
[the remark this answers is not in the document]
Comment (neildunlop) [reply .idea/misc.xml:1]: Done

File: android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/FlutterMapboxNavigationPlugin.kt [hunk 2]
```diff
@@ -55,7 +56,8 @@ public class FlutterMapboxNavigationPlugin : FlutterPlugin, MethodCallHandler,
 
         lateinit var routes: List<DirectionsRoute>
         private var currentRoute: DirectionsRoute? = null
-        val wayPoints: MutableList<Point> = mutableListOf()
+        //val wayPoints: MutableList<Point> = mutableListOf()
```
[the remark this answers is not in the document]
Comment (neildunlop) [reply android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/FlutterMapboxNavigationPlugin.kt]: Done

File: android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/models/SimpleWaypoint.kt (hunk 1 above)
Comment (neildunlop) [reply android/src/main/kotlin/com/eopeter/flutter_mapbox_navigation/models/SimpleWaypoint.kt]: Merged Waypoint and SimpleWaypoint into Waypoint

**Review by eopeter — APPROVED** [review]

**Pull request merged**